boxsdk/session/box_session.py

```diff
@@ -81,7 +82,7 @@ def __init__(self, oauth, network_layer, default_headers=None):
         """
         self._oauth = oauth
         self._network_layer = network_layer
-        self._default_headers = default_headers or {}
+        self._default_headers = default_headers or {'User-Agent': Version.USER_AGENT_STRING}
```

Comment:
I think `requests` fixed the bytestring problem many releases ago, and we require a version later than the fixed version.